source/includes/table-permissions-self-hosted.rst

```diff
@@ -4,47 +4,58 @@
 
 .. list-table::
    :header-rows: 1
+   :stub-columns: 1
+   :widths: 15 20 65
 
    * - Sync Type
-     - Target
+     - Cluster
      - Required Permissions
 
    * - default
      - source cluster
-     -
-
-         - readAnyDatabase
-         - backup
-         - clusterMonitor
+     - - :authrole:`readAnyDatabase`
+       - :authrole:`backup`
+       - :authrole:`clusterMonitor`
```

Comment:
**Suggestion**

Can we alphabetize each row for permissions, so its easier to compare / read?

Reply:
Fixed